cosmos/passage.json

```diff
@@ -0,0 +1,51 @@
+{
+  "rpc": "https://rpc.passage.vitwit.com",
+  "rest": "https://api.passage.vitwit.com",
+  "nodeProvider": {
+    "name": "Passage",
+    "email": "chain@passage.io"
+  },
+  "chainId": "passage-2",
+  "chainName": "Passage",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/passage/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "PASG",
+    "coinMinimalDenom": "upasg",
+    "coinDecimals": 6,
+    "coinGeckoId": "passage",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/passage/upasg.png"
+  },
+  "walletUrlForStaking": "https://resolute.vitwit.com/staking/passage",
```

Comment:
@mcianni Can you remove `walletUrlForStaking` field?

Reply:
@HeesungB Done!